tests/Drupal/Tests/Driver/Drupal7FieldHandlerTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Tests\Driver\Drupal7FieldHandlerTest
+ */
+
+namespace Drupal\Tests\Driver;
+
+/**
+ * Class Drupal7FieldHandlerTest
+ * @package Drupal\Tests\Driver
+ */
+class Drupal7FieldHandlerTest extends FieldHandlerAbstractTest {
+
+  /**
+   * @dataProvider dataProvider
+   */
+  public function testFieldHandlers($class_name, $entity, $entity_type, $field, $expected_values)
+  {
+    $handler = $this->getMockHandler($class_name, $entity, $entity_type, $field);
+
+    $field_name = $field['field_name'];
+    $expanded_values = $handler->expand($this->values($entity->$field_name));
+    $this->assertArraySubset($expected_values, $expanded_values);
+  }
+
+  /**
+   * Data provider.
+   *
+   * @return array
+   */
+  public function dataProvider()
+  {
+    return [
+
+      // Test default text field provided as simple text.
+      [
```

Comment:
For now, I'd like to maintain PHP 5.3 compatibility if we can--Drupal 7 still supports that.